TranslatorReasonerAPI.yaml

```diff
@@ -909,6 +917,7 @@ components:
             Edge attributes provided by this TRAPI web service.
           items:
             $ref: '#/components/schemas/MetaAttribute'
+          nullable: true
```

Comment:
```suggestion
          default: []
```

Reply:
What did we decide to do here? Do we allow MetaEdge.attributes to be "nullable: true" or do we switch to "default: []"?
I recall some spirited discussion, but the conclusion does not appear to be captured in the minutes.
I vaguely recall that there were more proponents of "nullable: true", but it is very hazy.

Reply:
We want to allow both null and empty, to distinguish between "I'm not telling you" and "I'm telling you there's nothing".